src/index.html

```diff
@@ -1,13 +1,369 @@
+                      alt="smart-home-01"
+                      class="category__photo"
+                      >
+                  </a>
+
+                  <a
+                  href="#"
+                  target="_blank"
+                  class="category__link"
+                  >
+                    <img
+                      src="images/products/smart-home-02.png"
+                      alt="smart-home-02"
+                      class="category__photo"
+                    >
+                  </a>
+                </div>
+
+                <h3 class="category__name">Smart Home</h3>
+              </article>
+              <div class="category__button">
+                <button
+                  type="submit" class="button button--view-all">
+                  View all
+                </button>
+              </div>
+            </div>
+
+          </section>
+        </div>
+      </div>
+      <div class="about-us">
+        <section class="about-us__story" id="about-us">
+
+          <h2 class="about-us__title">
+            Timeless, for 50 years and counting
+          </h2>
+
+          <p class="about-us__desctiption">
+            Bang & Olufsen is expanding its Recreated Classics Program with the launch of Beosystems – a limited edition music system that transcends time by bridging the gap between one of our iconic designs from 1972 and today’s cutting-edge digital technology.
+          </p>
+
+        </section>
+      </div>
+
+      <div class="container">
+        <div class="main__content">
+          <section class="contact-us">
+            <h2 class="contact-us__title">Contact us</h2>
+            <div class="contact-us__wrapper">
+              <form
+                  onsubmit="this.reset(); return false;"
+                  name="contact"
+                  class="contact-us__form"
+                  action="#"
+                  method="post"
+                >
+                <input
+                  class="contact-us__input contact-us--name"
+                  type="text"
+                  name="name"
+                  placeholder="Name"
+                  required
+                >
+                <input
+                  class="contact-us__input contact-us--email"
+                  type="email"
+                  name="email"
+                  placeholder="Email"
+                  required
+                >
+                <textarea
+                  class="contact-us__input contact-us--message"
+                  name="message"
+                  placeholder="Message"
+                  rows="30"
+                  cols="10"
+                  required
+              ></textarea>
+                <button class="button button--contact-us" type="submit">Send</button>
+              </form>
+              <div class="contact-us__contacts">
+                <h3 class="contacts__title">Phone</h3>
+                <a
+                  href="tel:+1 234 5555-55-55"
```

Comment:
Need to fix format phone everywhere only in href
```suggestion
                  href="tel:+123455555555"
```